### PR TITLE
UMD wrapper

### PR DESCRIPTION
**Pull request by jrit**
@Tathanen this is so this is CommonJS compatible. I thought it would be good to include the unminified built file, but should that still be excluded?

### CODE REVIEW

File: dist/vokal-ng-api.js [hunk 1]
```diff
@@ -0,0 +1,365 @@
+(function (root, factory) {
```
Comment (Tathanen) [inline dist/vokal-ng-api.js]: Does this whole chunk need to be so coding-style-incompatible?

Comment (Tathanen) [inline dist/vokal-ng-api.js]: Oh wait it's being generated by that UMD thing aint't it.

**Comment by Tathanen** [comment]
I don't have a super-strong opinion on including the unminified file.  I guess it could be useful for debugging purposes, getting more meaningful line errors and the like.

**Comment by Tathanen** [comment]
In `package.json` the test command includes the uglify task, the assumed workflow before is that you'd run the tests before opening a PR and that would generate your built files, to make sure they're updated for the PR.  Go ahead and update that line to use `grunt default` instead, I think that sounds right.

File: dist/vokal-ng-api.js (hunk 1 above)
Comment (jrit) [inline dist/vokal-ng-api.js]: Yeah, that whole chunk at the top is what grunt-umd creates, which is why it is in dist but not source

**Comment by jrit** [comment]
Finally got the build passing on Travis, so should be good to merge.

**Comment by Tathanen** [comment]
LGTM

Are we using Travis for things now?

**Comment by dmaloneycalu** [comment]
Travis v. Drone?

**Comment by jrit** [comment]
Travis is for public projects, otherwise only people at Vokal can see build output. The travis.yml is really simple, as you can see in this repo.

**Pull request merged**